git-hg/diff.sh

```diff
@@ -65,7 +77,10 @@ function updateMercurialClone() {
 }
 
 function runDiff() {
-  diffNum=$(diff -rq openjdk-git openjdk-hg -x '.git' -x '.hg' -x '.hgtags' -x '.hgignore' -x 'get_source.sh' -x 'README.md' | wc -l)
+
+  diff -rq openjdk-git openjdk-hg -x '.git' -x '.hg' -x '.hgtags' -x '.hgignore' -x 'get_source.sh' -x 'README.md' > changes.diff
+
+  diffNum=$(wc -l < changes.diff)
 
   if [ "$diffNum" -gt 0 ]; then
```

Comment:
Do we want to log somewhere that the `changes.diff` file can be inspected?

Do we need to delete that file later?

Reply:
can archive it in the job config